File: 2696.js
var minLength = function(s) {
  while(s.includes("AB") || s.includes("CD")){
    s = s.replace("AB","");
    s = s.replace("CD","");
  }
  return  s.length;
};

console.log(minLength(s = "ABFCACDB"));

File: 2696.cpp
#include <iostream>
#include <string>
using namespace std;

int main()
{
  string s = "ABFCACDB";
  while (s.find("AB") != string::npos || s.find("CD") != string::npos)
  {
    if (s.find("AB") != string::npos)
    {
      s = s.replace(s.find("AB"), 2, "");
    }
    if (s.find("CD") != string::npos)
    {
      s = s.replace(s.find("CD"), 2, "");
    }
  }
  return s.length();

  return 0;
}
